charts/bpa/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: bpa
 description: The Business Partner Agent allows to manage and exchange master data between organizations.
 type: application
 
-version: 0.7.1
+version: 0.7.2
```

Comment:
this would be a 0.8-something (pre-release for 0.8). what you used is for fixes.